freeipa/src/main/resources/freeipa-salt/salt/freeipa/templates/ipa-rewrite.conf.j2

```diff
@@ -0,0 +1,31 @@
+# VERSION 6 - DO NOT REMOVE THIS LINE
```

Comment:
Can you add a comment at the top saying "Create by Cloudbreak salt install" or something like that?